packages/docs/docs/providers/saml-jackson.md

```diff
@@ -0,0 +1,58 @@
+---
+id: saml-jackson
+title: BoxyHQ SAML Jackson
+---
+
+## Documentation
+
+BoxyHQ SAML Jackson is an open source service that handles the SAML login flow as an OAuth 2.0 flow, abstracting away all the complexities of the SAML protocol.
+
+You can deploy SAML Jackson as a separate service or embed it into your app using our NPM library. [Please check our documentation for more details](https://boxyhq.com/docs/jackson/deploy)
+
+## Configuration
+
+SAML login requires a configuration for every tenant of yours. One common method is to use the domain for an email address to figure out which tenant they belong to. You can also use a unique tenant ID (string) from your backend for this, typically some kind of account or organization ID.
+
+Check our [documentation](https://boxyhq.com/docs/jackson/getting-started#2-saml-config-api) for more details.
+
+## Options
+
+The **SAML Jackson Provider** comes with a set of default options:
+
+- [BoxyHQ Provider options](https://github.com/nextauthjs/next-auth/blob/main/src/providers/boxyhq.ts)
+
+You can override any of the options to suit your own use case.
+
+## Example
+
+```ts
+import SAMLJacksonProvider from "next-auth/providers/saml-jackson"
+...
+providers: [
+  SAMLJacksonProvider({
+    issuer: "http://localhost:5000",
+    clientId: "dummy", // The dummy here is necessary since we'll pass tenant and product custom attributes in the client code
+    clientSecret: "dummy", // The dummy here is necessary since we'll pass tenant and product custom attributes in the client code
```

Comment:
```suggestion
    issuer: "http://localhost:5000",
    clientId: "dummy", // The dummy here is necessary since we'll pass tenant and product custom attributes in the client code
    clientSecret: "dummy", // The dummy here is necessary since we'll pass tenant and product custom attributes in the client code
```

This is an unusual flow, so I would like to have more context here.

If the `clientId` and `clientSecret` values aren't important, we should just omit them, or set them by default in the provider code.